index.html

```diff
@@ -267,7 +267,7 @@ <h2>Skills</h2>
         <div class="contact-div">
           <h1>Contact me</h1>
           <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-          <form class="contact-form" action="" method="post">
+          <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
             <input type="text" name="name" id="name" placeholder="Name" maxlength="30" required>
             <input type="email" name="email" id="email" placeholder="Email" required>
             <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500"></textarea>
```

Comment:
- [ ] As stated the project requirement A user cannot submit the form if one or more of the fields is empty, So Kindly I recommend you to add for message input fields to have the `required` attribute to activate the validation on submit.